runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcWindmillServerTest.java

```diff
@@ -21,6 +21,7 @@
 
 import java.io.InputStream;
 import java.io.SequenceInputStream;
+import java.lang.reflect.Array;
```

Comment:
Is this used anywhere?

Reply:
Good point, removed, (this also fails one of the GitHub actions)